src/status_im/ui/screens/wallet/collectibles/superrare/views.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im.ui.screens.wallet.collectibles.superrare.views
+  (:require [re-frame.core :as re-frame]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.action-button.action-button :as action-button]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.screens.wallet.collectibles.styles :as styles]
+            [status-im.ui.screens.wallet.collectibles.views :as collectibles]))
+
+(defmethod collectibles/render-collectible :SUPR [_ {tokenId :tokenId {:keys [description name imageUri] :as m} :metadata}]
+  [react/view {:style styles/details}
+   [react/view {:style styles/details-text}
+    [react/image {:style  (merge {:resize-mode :contain :width 100 :height 100} styles/details-image)
+                  :source {:uri imageUri
+                           :k   1.4}}]
+    [react/view {:flex 1 :justify-content :center}
```

Comment:
move to styles? ¯\_(ツ)_/¯

Reply:
aren't we wasting energy creating dedicated style namespace for every views namespace, having to come up with names for styles, and in the end never reuse again because it is so specific ? 
personally I would very much appreciate if we were doing the styling in-line, but ofc it is not always that easy and sometimes you have the defstyles macro for platform specific stuff so I don't have a strong opinion about it. But I sometime find it wasteful to follow the pattern that we have right now